app/cinema/hall.py

```diff
@@ -0,0 +1,18 @@
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list[Customer],
+            cleaning_staff: Cleaner) -> None:
+        print(f'"{movie_name}" started in hall number {self.number}.')
+        for i in range(len(customers)):
```

Comment:
```suggestion
        for customer in customers:
```
u can use such cycle - it's more clear as for me